nethserver-ejabberd.spec

```diff
@@ -69,7 +69,7 @@ mkdir -p %{buildroot}/%{_localstatedir}/lib/nethserver/ejabberd/upload
 %systemd_preun ejabberd.service
 
 %postun
-%systemd_postun
+%systemd_postun ejabberd.service
```

Comment:
Could you post the error message related to this change?

I'd like to split this change in a separate PR :thinking:

Reply:
this is the error

```
╰─➤  make-rpms *.spec
make-rpms [INFO] git describe: 1.8.2 2 g62fa10f
error: The %systemd_postun macro requires some arguments
Traceback (most recent call last):
  File "/usr/bin/spectool", line 441, in <module>
    exit(main())
  File "/usr/bin/spectool", line 366, in main
    spec = Spec(path)
  File "/usr/bin/spectool", line 207, in __init__
    self.spec = rpm.spec(self.path)
ValueError: can't parse specfile

error: The %systemd_postun macro requires some arguments
Traceback (most recent call last):
  File "/usr/bin/spectool", line 441, in <module>
    exit(main())
  File "/usr/bin/spectool", line 366, in main
    spec = Spec(path)
  File "/usr/bin/spectool", line 207, in __init__
    self.spec = rpm.spec(self.path)
ValueError: can't parse specfile

basename: missing operand
Try 'basename --help' for more information.
Cleaning up temporary files..
removed './.tmpO2liP99.spec'
removed './.tmpWQleDJI.gitlog'
```

It seems something new, but I am sure it makes sense

Reply:
Proposed change seems good to me, see https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_systemd

I'm ok for moving to another PR

Reply:
this has been moved to https://github.com/NethServer/nethserver-ejabberd/pull/38